sample/src/main/java/com/google/modernstorage/sample/filesystem/FileSystemFragment.kt

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.modernstorage.sample.filesystem
+
+import android.os.Bundle
+import android.text.format.Formatter
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.activity.result.contract.ActivityResultContracts.OpenDocument
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.viewModels
+import com.google.android.material.snackbar.Snackbar
+import com.google.modernstorage.sample.R
+import com.google.modernstorage.sample.databinding.FragmentFilesystemBinding
+
+class FileSystemFragment : Fragment() {
```

Comment:
Comment about this class (and probably calling out that it's only 26+, along with a `@RequiresApi(Build.VERSION_CODES.O)` annotation.